### DISCUSSION TITLE
Merge Albums

### DISCUSSION THREAD

**Discussion by Elf3874** [ente-io/ente#3133] [Enhancements]
Though not being a big feature, its one I think could have a place.
![ente merge concept](https://github.com/user-attachments/assets/0ac456b8-68d5-4d78-91d9-a7f4cd750515)

![ente merge concept 2](https://github.com/user-attachments/assets/9b1e5be0-cf35-4c2b-8013-42dce27b5a68)

**Comment by Trac3rTong** [comment]
I fully support this request. When importing photos into Ente, all subfolders are turned into separate albums, which quickly becomes confusing if the parent folder structure is lost.

For example, if you import a folder from a two-week trip where each day was stored in a separate subfolder, you suddenly end up with 14 separate albums on Ente – all at the same level, without any indication that they belong together.
Since Ente doesn't currently support album trees or nested albums, it would be really helpful to at least have a simple way to merge multiple albums back into one. This would help keep things organized until (hopefully) real nested albums are supported in the future.
